virttest/qemu_devices/qcontainer.py

```diff
@@ -315,6 +359,44 @@ def allocate_iothread(self, iothread, device):
             )
             raise TypeError(err_msg)
 
+    def allocate_iothread_vq(self, iothread, device):
+        """
+        Allocate iothread( supporting vq ) for device to use.
+
+        :param iothread: iothread specified in params
+                         could be:
+                         'auto': allocate iothread based on schems specified by
+                                 'iothread_scheme'.
+                         iothread id: request specific iothread to use.
+        :param device: device object
+        :return: iothread object allocated
+        """
+        if self.is_dev_iothread_vq_supported(device):
+            iothreads = self.__iothread_manager.request_iothread(iothread)
```

Comment:
use self.iothread_manager? it works all right, maybe it's better for code reading, as we always use self.iothread_manager anywhere except this one.

Reply:
I am OK to keep this. 
Just a little bit curious about the difference between this line and line 389

Reply:
> I am OK to keep this. Just a little bit curious about the difference between this line and line 389

Got you concern, after checking the codes, I found the previous codes called `__iothread_manager` by the previous authors. However, let's keep our style `iothread_manager` since we created the property `iothread_manager`.  The Line#375 has been changed.